packages/core/test/render3/imported_renderer2.ts

```diff
@@ -39,8 +39,9 @@ export class SimpleDomEventsPlugin extends EventManagerPlugin {
 export function getRendererFactory2(document: any): RendererFactory2 {
   const fakeNgZone: NgZone = new NoopNgZone();
   const eventManager = new EventManager([new SimpleDomEventsPlugin(document)], fakeNgZone);
+  const appId = 'dummyappid';
```

Comment:
Nit: Can we use a different word from "dummy" here? It's a term that has a history as an ableist slur and we should probably avoid including it in code when there's other options.

Thanks!!!

```suggestion
  const appId = 'placeholderappid';
```